apps/storybook/components/funke/FirstLaunchScreen.stories.tsx

```diff
@@ -12,42 +11,26 @@ const FirstLaunchScreen = () => {
         flex={1}
         width="100%"
         height="100%"
-        colors={['#EFF3F6', '#FFFFFF']}
+        colors={['$grey-100', '$white']}
         start={[0.5, 0]}
         end={[0.5, 1]}
       />
       <YStack p="$4" gap="$4" flex-1 justifyContent="space-between">
-        {/* This stack keeps the  */}
+        {/* This stack ensures the right spacing  */}
         <YStack flex={3} />
         <YStack gap="$2">
-          {/* todo: grey-5 */}
-          <Heading variant="h1" color="#282C37">
-            Ausweis Wallet
-          </Heading>
-          {/* todo: 1 is a bit too small, maybe just hardcode */}
-          <Separator borderWidth="$1" borderRadius="$1" borderColor="$primary-500" width="$3" />
-          {/* todo: grey-4 */}
-          <Heading variant="h1" color="#656974">
+          <Heading variant="title">Ausweis Wallet</Heading>
+          <Separator borderWidth={3} borderRadius={3} borderColor="$primary-500" width="$4" />
```

Comment:
Can you use token values for the borderWidth and radius as well? Or did they not work.

"$1" or "$1.5" maybe.

```suggestion
          <Separator borderWidth="$1" borderRadius="$1" borderColor="$primary-500" width="$4" />
```

Reply:
Yes, but I think not everyone should be based on the size values right? as it will just look weird when the value of $1 changes

Reply:
Ah yes you're right. For borders I often used fixed values as they don't need to change ever.